pyxrf/model/draw_image_rgb.py

```diff
@@ -290,17 +309,69 @@ def _compute_equal_axes_ranges(x_min, x_max, y_min, y_max):
 
             return x_axis_min, x_axis_max, y_axis_min, y_axis_max
 
-        # Set equal ranges for the axes data
-        yd, xd = selected_data.shape[1], selected_data.shape[2]
-        xd_min, xd_max, yd_min, yd_max = 0, xd, 0, yd
-        # Select minimum range for data
-        if (yd <= math.floor(xd / 100)) and (xd >= 200):
-            yd_min, yd_max = -math.floor(xd / 200), math.ceil(xd / 200)
-        if (xd <= math.floor(yd / 100)) and (yd >= 200):
-            xd_min, xd_max = -math.floor(yd / 200), math.ceil(yd / 200)
+        def _adjust_data_range__min_ratio(c_min, c_max, c_axis_range, *, min_ratio=0.01):
+            """
+            Adjust the range for plotted data along one axis (x or y). The adjusted range is
+            applied to the 'extend' attribute of imshow(). The adjusted range is always greater
```

Comment:
https://matplotlib.org/3.1.0/api/_as_gen/matplotlib.pyplot.imshow.html
```suggestion
            applied to the 'extent' attribute of imshow(). The adjusted range is always greater
```